.circleci/config.yml

```diff
@@ -234,7 +234,7 @@ workflows:
             container=offenderapi,name=FEATURES_NOMS_UPDATE_BOOKING_NUMBER,                         value=true,
             container=offenderapi,name=FEATURES_NOMS_UPDATE_KEYDATES,                               value=true,
             container=offenderapi,name=FEATURES_NOMS_UPDATE_NOMS_NUMBER,                            value=true,
-            container=offenderapi,name=FEATURES_APPLY_LIMITED_ACCESS_MARKERS,                       value=false,
+            container=offenderapi,name=FEATURES_APPLY_LIMITED_ACCESS_MARKERS,                       value=true,
```

Comment:
Are you happy for this to be on now, since there will probably be a prod deployment with this in within the next 24 hours?

Reply:
We do want it out ASAP but we've got some QA to do so probably not wise to tie ourselves to others' releases. Have backed it out for now, good call.